schemas/com.iterable/system_webhook/jsonschema/1-0-0

```diff
@@ -0,0 +1,695 @@
+          "maxLength": 4096
+        },
+        "campaignId": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "minimum": 0,
+          "maximum": 9223372036854775807,
+          "description": "Reference to the campaign from which the event originated (e.g., 74768)"
+        },
+        "campaignName": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Campaign name",
+          "maxLength": 4096
+        },
+        "canonicalUrlId": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Reference to the URL associated with the event (e.g., '3145668988')",
+          "maxLength": 4096
+        },
+        "catalogCollectionCount": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "minimum": 0,
+          "maximum": 2147483647,
+          "description": "The number of times a catalog collection was referenced in the message."
+        },
+        "catalogLookupCount": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "minimum": 0,
+          "maximum": 2147483647,
+          "description": "The number of times a catalog was referenced in the message."
+        },
+        "channelId": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "minimum": 0,
+          "maximum": 9223372036854775807,
+          "description": "Unique identifier of the channel (e.g., 2203)"
+        },
+        "channelIds": {
+          "type": [
+            "array",
+            "null"
+          ],
+          "items": {
+            "type": [
+              "integer",
+              "null"
+            ],
+            "minimum": 0,
+            "maximum": 9223372036854775807
+          },
+          "description": "Channel IDs"
+        },
+        "city": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "City (e.g., 'San Jose')",
+          "maxLength": 4096
+        },
+        "contentAvailable": {
+          "type": [
+            "boolean",
+            "null"
+          ],
+          "description": "Is content available"
+        },
+        "contentId": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "minimum": 0,
+          "maximum": 9223372036854775807,
+          "description": "Reference to the content the event is associated with (e.g., 3681)"
+        },
+        "country": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Country (e.g., 'United States')",
+          "maxLength": 4096
+        },
+        "createdAt": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Creation timestamp (e.g., '2017-05-15 23:59:47 +00:00')",
```

Comment:
It makes me sad that iterable send timestamps in this format.  This will end up getting stored as string type in the warehouse.

If they sent them in iso 8601 format then you could put `format: date-time` in the schema, and then warehouses could store it as timestamp type.

But I understand this is something you cannot control.